### ISSUE TITLE
Adblock Warning Removal List is blocked by uBlock Origin since version 1.29.2

### ISSUE THREAD

**Issue by TonyTough** [LiCybora/NanoDefenderFirefox#185]
uBlock Origin intentionally blocks Adblock Warning Removal List since version 1.29.2:
https://www.reddit.com/r/uBlockOrigin/comments/iruupe/latest_ublock_origin_beta_erased_my_subscribed/

According to the installation instructions for Nano Defender, Adblock Warning Removal List is required for Nano Defender to work:
https://jspenguin2017.github.io/uBlockProtector/

So what does this mean for users of Nano Defender?
Does Nano Defender work without Adblock Warning Removal List without problems?

**Comment by hawkeye116477** [comment]
https://github.com/NanoAdblocker/NanoCore/issues/359

**Comment by JackSlaterIV** [comment]
I was going to open a new issue about this matter after switching from Nano Adblocker to Ublock origin (due to the handover concerns).
I did not understand if the ‘Adblock Warning Removal List’ is still needed for NanoDefender to work correctly.
Anyway, this page is linked from NanoDefender and is still showing it as required: https://jspenguin2017.github.io/uBlockProtector/#extra-installation-steps-for-ublock-origin.
This may be confusing for the users switching to Ublock these days.

**Comment by LiCybora** [comment]
You may keep this first unless you experience issue, and report to me then. I will announce whether it is still needed when new version comes out.

I need some time to analyze the effectiveness of this filter to Nano Defender.

**Comment by xelaseer** [comment]
@LiCybora I haven't independently verified this, but according to [this](https://github.com/uBlockOrigin/uBlock-issues/issues/951#issuecomment-602799609), this filter is now included within the RU Adblock filter list.

Should be easy enough to verify. I might try to do so if I have time.

**Comment by beppe9000** [comment]
As a workaround I solved this by using the **import list** feature of ublock origin with this mirror url here: https://pastebin.com/raw/fXWUXTVb
